src/app/zap-templates/zcl/zcl.json

```diff
@@ -729,44 +729,7 @@
             "MinViewport",
             "MicrophoneCapabilities",
             "SpeakerCapabilities",
-            "Viewport",
-            "MaxConcurrentEncoders",
```

Comment:
Per the other comments: should leave the scalar bits here, so we don't generate useless code and unnecessary memory allocations for them.  Just remove the lists.